### PR TITLE
Real Estate 2008

### PR DESCRIPTION
**Pull request by Oxalisviolacea**
Hanna Davis

### CODE REVIEW

**Comment by Oxalisviolacea** [comment]
This PR:
- creates room class
-creates house class
- iteration 4 tests are not passing

File: lib/house.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
```
Comment (mikedao) [inline lib/house.rb:5]: Check the spec - this is not what the spec is asking for,

File: lib/room.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Room
+  attr_reader :category
+
```
Comment (mikedao) [inline lib/room.rb:3]: Missing is_painted? and paint methods here.

File: lib/house.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+class House
```
Comment (mikedao) [inline lib/house.rb:1]: Missing #details method

**Pull request closed without merging**